packages/i18nify-js/README.md

```diff
@@ -541,6 +543,48 @@ console.log(getDialCodeByCountryCode('BR')); // Outputs the dial code for Brazil
 console.log(getDialCodeByCountryCode('DE')); // Outputs the dial code for Germany (+49)
 ```
 
+#### getMaskedPhoneNumber(options)
+
+📞🔒 The getMaskedPhoneNumber function is a versatile tool designed to handle phone number formatting and masking based on the specific requirements of different countries. This function is ideal for applications that require the display of partially hidden phone numbers for security purposes or privacy concerns. It supports a wide range of configurations, including options to mask portions of the phone number, specify the number of digits to mask, and choose whether to mask digits from the beginning or end of the number.
+
+##### Examples
+
+```javascript
+// Masking a U.S. phone number completely
+console.log(
+  getMaskedPhoneNumber({
+    countryCode: 'US',
+    phoneNumber: '2025550125',
+    withDialCode: true,
+  }),
+);
+// Output: +1 xxx-xxx-xxxx
+
+// Partially masking an Indian phone number, hiding the last 6 digits
+console.log(
+  getMaskedPhoneNumber({
+    countryCode: 'IN',
+    phoneNumber: '9876543210',
+    maskingOptions: {
+      completeMasking: false,
+      prefixMasking: false,
+      maskedDigitsCount: 6,
+      maskingChar: '*',
```

Comment:
looks good, but isn't this a little confusing ?
what does prefixMasking means ?
I was thinking to have one key which can have following values
`full, prefix-half, suffix-half, alternate`  
   maskedDigitsCount and maskingChar can stay as it is
   thoughts ? @RgnDunes

Reply:
`prefixMasking` means masking starting digits of phone number. In case `prefixMasking` is false last digits of phone number will be masked. 

What is alternate in your example @tarun-khanna ?

Reply:
Discussed over call.